cg/store/crud/create.py

```diff
@@ -398,12 +397,13 @@ def add_organism(
             **kwargs,
         )
 
-    def add_order(self, customer: Customer, ticket_id: int) -> Order:
+    def add_order(self, customer: Customer, ticket_id: int, **kwargs) -> Order:
         """Build a new Order record."""
         order = Order(
             customer=customer,
             order_date=datetime.now(),
             ticket_id=ticket_id,
+            **kwargs,
         )
         return order
```

Comment:
this only creates an order object but does not add it to the DB. this misses the self.session.add()

Reply:
I think all our Create functions return only the object without adding them to the transaction. So I tried to make this function as similar as the others. We should discuss if we want these functions to add to the transaction too.